app/assets/javascripts/discourse/app/templates/components/user-card-contents.hbs

```diff
@@ -73,6 +73,10 @@
               label="user.private_message"}}
             </li>
           {{/if}}
+          {{plugin-outlet
+          name="user-card-chat-button" connectorTagName="li"
```

Comment:
```suggestion
          name="user-card-below-message-button" connectorTagName="li"
```

It's important to give this a generic name, in case a non-chat plugin/component wants to use this outlet too.